GameOfLifeApp/Controllers/ApplicationManager.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace GameOfLifeApp
+{
+    public class ApplicationManager : IApplicationManager
+    {
+        private ICommunicator input;
+        private IGameLogic data;
+        private ISerializer dataBase;
+        private IGameManager manager;
+        private IGameField pr;
```

Comment:
why do you need `private IGameField pr;` here?

Reply:
Removed.